kodilla-testing/src/main/java/com/kodilla/testing/weather/stub/WeatherForecast.java

```diff
@@ -13,13 +13,65 @@ public WeatherForecast(Temperatures temperatures) {
     public Map<String, Double> calculateForecast() {
         Map<String, Double> resultMap = new HashMap<>();
 
-        for (Map.Entry<String, Double> temperature :
-                temperatures.getTemperatures().entrySet()) {
-
-            // adding 1 celsius degree to current value
-            // as a temporary weather forecast
-            resultMap.put(temperature.getKey(), temperature.getValue() + 1.0); // [1]
+        for (Map.Entry<String, Double> temperature : temperatures.getTemperatures().entrySet()) {
+            resultMap.put(temperature.getKey(), temperature.getValue() + 1.0);
         }
         return resultMap;
     }
+
+    public Double calculateAverageTemperature() {
+
+        double result = 0;
+        if (temperatures.getTemperatures().size() == 0) {
```

Comment:
Return Double.NaN

Reply:
zmieniłem z return null na return Double.NaN. Tylko nie wiem dlaczego. Co to zmieni?

Reply:
Typ Double ma cos takiwgo jak NaN - not a number. Nic to mie zmienia poza faktem, ze nie mamy nulla